src/components/templates/PosterPage/components/ShareModal/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { FACEBOOK_APP_ID } from "secrets";
+import { PosterPageVenue } from "types/venues";
+import { WithId } from "utils/id";
+
+const getCategoriesFromVenue = (venue: WithId<PosterPageVenue>) => {
+  return venue?.poster?.categories || [];
+};
+
+export const getTitleTextForSharing = (
+  venue: WithId<PosterPageVenue>,
+  url: string
+) => {
+  if (!venue.poster) return "Check out this OHBM Poster";
+
+  return `Check out this OHBM Poster, ${venue.poster.title} by ${venue.poster.authorName} at ${url}`;
```

Comment:
We don't want OHBM references hardcoded into this PR (as I have called out in previous reviews)

There is a `const` in `src/settings` that is used to set the window title, and is the 'main event name' or similar. We could potentially use that (though it's hardcoded per environment)

Or we could just remove it entirely and have the text as "Check out this poster"

Reply:
Or we could just remove it entirely and have the text as "Check out this poster", which is probably my preference/the simplest/most robust solution here.

Reply:
What do you think of this as a solution @mike-lvov? Uses `PLATFORM_BRAND_NAME` (which defaults to `Sparkle` on most environments, and is `OHBM2021` in `env/ohbm`), and seems it would read well in both situations/generally.

```suggestion
  if (!venue.poster) return `Check out this poster on ${PLATFORM_BRAND_NAME}`;

  return `Check out this poster on ${PLATFORM_BRAND_NAME}! ${venue.poster.title} by ${venue.poster.authorName} at ${url}`;
```

Reply:
Though another question, what is the point of this line?
```
if (!venue.poster) return `Check out this poster on ${PLATFORM_BRAND_NAME}`;
```

I'm guessing it's guarding against the case when `venue.poster` is `undefined`.. but it will end up making a pointless social media share since it's not able to include any details about the poster. We might be better with just a straight `if (!vnue.poster) return` here, and then handling that case of `undefined` in a way that we communicate the error to the user/don't allow it to post/etc.